### DISCUSSION TITLE
Remote control design

### DISCUSSION THREAD

**Discussion by edutsilva** [GerryDazoo/Slinger#214] [General]
Hi,
Is there a way to custom design the remote control? I do not mean the remote codes. I want it to be more cool.

Thank you.

**Comment by GerryDazoo** [comment]
Please read the latest release notes.

**Reply by edutsilva** [reply]
Thank you!

**Reply by GerryDazoo** [reply]
Check also CustomRemotes. There's an .html example there.

**Reply by edutsilva** [reply]
Thank you. Just download it. One newbie question: does the html file replaces the remote.txt file or they are both used?

**Reply by pfuller88** [reply]
FYI... I just updated the HTML file with better code, see my recent post.  Also a black background.   My understanding is the HMTL replaces the need for the TXT file.

**Reply by GerryDazoo** [reply]
Yes, that is correct.

**Comment by GerryDazoo** [comment]
Check this out. https://github.com/GerryDazoo/Slinger/discussions/224

**Comment by pfuller88** [comment]
Nice.   Did you see my latest version, updated yesterday, screenshot below.   Getting closer.  The FIOS design is nice, gave me some ideas.
